docs/pages/connect-your-client/putty-winscp.mdx

```diff
@@ -1,16 +1,22 @@
 ---
-title: Using PuTTY to connect to SSH nodes via Teleport
-description: This reference shows you how to use PuTTY to connect to SSH nodes through Teleport
+title: Using PuTTY to connect to SSH nodes and WinSCP to transfer files through Teleport
+description: This reference shows you how to use PuTTY to connect to SSH nodes and WinSCP to transfer files through Teleport
 ---
 
 This guide will show you how to use the Teleport client tool `tsh` to add saved sessions for use
 with [PuTTY](https://www.putty.org/), and then how to use PuTTY as a client to connect to SSH nodes.
 
+It will also show you how to optionally use these saved sessions with [WinSCP](https://winscp.net) to transfer
+files from SSH nodes using SFTP.
+
 You will learn how to:
 
 - Generate saved PuTTY sessions for SSH nodes attached to a Teleport cluster.
 - Log in to an interactive shell using these saved PuTTY sessions.
 
+You will also optionally learn how to:
+- Import a saved PuTTY session into WinSCP, then start an SFTP session to transfer files from an SSH node.
```

Comment:
Personally, I would just merge this bullet with the previous list. A list of 1 item doesn't need to be a list, and I'm not sure what "optionally learn" means.